git-ftp

```diff
@@ -789,10 +789,12 @@ submodule_catchup() {
 	set_submodule_args
 	url="$(git config git-ftp.url)"
 	print_info "Submodules are $GIT_SUBMODULES"
-	for submodule in "${GIT_SUBMODULES[@]}"
+	for submodule in ${GIT_SUBMODULES[@]}
 	do
 		print_info "Catching up submodule $submodule."
+		local module="$(pwd)"
```

Comment:
Good catch, this allows for nested submodules. `current_path` is probably a better name than `module` here. And I'm wondering if it would be even cleaner to use a sub-shell instead of going back. These three lines could be shortened to:
```bash
(cd "${SYNCROOT}$submodule" && exec "$0" "$ACTION" "${args[@]}" "$REMOTE_PROTOCOL://$REMOTE_HOST/${REMOTE_PATH}$submodule")
```

I guess, I haven't tested this.